packages/components/package.json

```diff
@@ -22,7 +22,7 @@
   "scripts": {
     "build": "npm run build:clean && npm run build:types && npm run build:js",
     "build:clean": "rm -rf dist/",
-    "build:js": "babel src -d dist --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/*.spec.* --ignore src/**/*.stories.*",
+    "build:js": "babel src -d dist --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/*.spec.* --ignore src/**/*.stories.* --ignore src/**/*.d.ts",
```

Comment:
Were we running into issues with babel processing .d.ts files?

I ask because it seems (without understanding deeply the build system here :smile:) slightly odd, since I would expect Babel to produce these files in the first place. Is it an issue because they're left around from subsequent builds?

Or am I misunderstanding everything?

Reply:
it's only because I added the `twin.d.ts` file 😞 I might remove from this one and add it to the Typography PR where I'm actually using `styled`

Reply:
oh lol I put it in the wrong spot 🤦‍♀️ either way, moving it out of this one